libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route/ReplayRouteSession.kt

```diff
@@ -139,7 +138,6 @@ class ReplayRouteSession : MapboxNavigationObserver {
         mapboxNavigation.mapboxReplayer.unregisterObserver(replayEventsObserver)
         mapboxNavigation.mapboxReplayer.stop()
         mapboxNavigation.mapboxReplayer.clearEvents()
-        mapboxNavigation.stopTripSession()
```

Comment:
Is it a logical breaking change for the ones who already integrated ReplayRouteSession? Replay session used to leave trip session state in stopped state and not it's running. 
I understand that it's an experimental API, so maybe it's worth to mention it in the changelog?

Reply:
The `stopTripSession` made it so downstream developers would call `startTripSession` after this call. So they will have a no-op. Most of the issues developers have had is due to the rapid session changes (observing trip state observer). Forcing them to introduce logic like this [StateResetController](https://github.com/mapbox/mapbox-navigation-android/blob/61415ea7ee81831b69fd698b0522bfd69119dd23/libnavui-app/src/main/java/com/mapbox/navigation/ui/app/internal/controller/StateResetController.kt#L12-L43).

The issue that it could introduce, is if someone is relying on `RouteTripSession.onDetached` to `stopTripSession`. Their trip session will remain started. The changelog speaks of this change so I'm not sure how an extra changelog would help?

Reply:
I've added a :warning: